styles.css

```diff
@@ -494,3 +506,245 @@ ul {
 #social-media-links #contact-links2 li {
   margin: 0 0.5rem;
 }
+
+#desk-img {
+  display: none;
+}
+
+#contact-form1 {
+  display: none;
+}
+
+#button1 {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  html {
+    font-size: 62.5%;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  body {
+    margin: 0;
+    font-size: 1.6rem;
+  }
+
+  #navbar ul {
+    display: flex;
+    margin-right: 41.3%;
+    width: 17.7%;
+    gap: 24px;
+    margin-top: 0;
+  }
+
+  .menu-3 li a {
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  #contact-links {
+    width: 50%;
+  }
+
+  #social-media-links #contact-links2 li {
+    margin: 0 0.5rem;
+  }
+
+  #contact-email {
+    display: none;
+  }
+
+  .about-me {
+    max-width: 88%;
+    position: relative;
+  }
+
+  #projects {
+    display: grid;
+    gap: 3.2rem 2.4rem;
+    grid-template-columns: 1fr 1fr 1fr;
+    padding: 0 14.2rem;
+  }
+
+  .project-cards {
+    border: 1px solid #ebf0ee;
+  }
+
+  .right-image {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+
+  .left-image {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+
+  #about-header {
+    text-transform: capitalize;
+    letter-spacing: 0.3px;
+    margin: 0;
+    padding: 19.2rem 49.6rem 2.4rem;
+    display: flex;
+    justify-content: center;
+  }
+
+  #about-text {
+    max-width: 92rem;
+    min-height: 9.6rem;
+    height: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 26rem;
+    margin-left: 290px;
+  }
+
+  .form-text {
+    max-width: 40%;
+    height: 189px;
+    font-family: inter, Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+    text-align: center;
+    color: #42526e;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .hide {
+    display: block;
+  }
+
+  .svg-img {
+    position: absolute;
+    z-index: -1;
+    min-width: 149rem;
+    max-height: min-content;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .wrapper {
+    display: flex;
+    gap: 24px;
+    flex-wrap: nowrap;
+    margin-top: 164px;
+    width: 100%;
+    margin-bottom: 237px;
+    margin-left: 5%;
+    background-image: url(/images/lastdude.png);
```

Comment:
- [ ] To show the image in the contact-me form please change this line to this 👇
```
background-image: url("images/lastdude.png");
```
- [ ] As for the images in the `About me section`, they are also not present because you have a `/` before `images` in the src👇 
```
<!--about section-->
<section id="about">
  <div class="about-me">
    <img class="right-image" `src="/images/bkg1.png"` alt="pic">
    <h3 id="about-header">About me</h3>
    <p id="about-text">
      I can help you build a product , feature or website Look through some
      of my work and experience! If you like what you see and have a project
      you need coded, don't hestiate to contact me.
    </p>
    <img class="left-image" `src="/images/bkg2.png"` alt="pic">
  </div>
  <div class="about-button">
    <button type="button">Get My Resume</button>
  </div>
```
To fix it you need to use `"images/bkg1.png"` and `"images/bkg2.png"` but please add the images as a `background image` property in the CSS file not as an `img` tag in the HTML file.
You can check this [link](https://www.w3schools.com/cssref/pr_background-image.asp) if you need to understand image paths better.